### PR TITLE
Remove code coverage for Sacred config scripts

### PR DESCRIPTION
**Pull request by shwang**
Code coverage checks currently fail whenever we add a new
Sacred config or namedconfig function because the body of these
functions is never meant to be executed. This patch excludes the
files containing these functions from CodeCov checks.

(Example of failure: #298)

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ coverage:
         paths:
           - "src/imitation/envs/examples/"
           - "src/imitation/scripts/"
+          - "!src/imitation/scripts/config"
```
Comment (AdamGleave) [inline .codecov.yml:15]: Why do we need to remove it here? The target is fixed at 0% for auxiliary so it'll never fail the check. Seems useful to still at least calculate the code coverage change.

**Review by AdamGleave — CHANGES_REQUESTED** [review]

File: .codecov.yml (hunk 1 above)
Comment (shwang) [reply .codecov.yml:15]: Ah, that's right. I was confusing it with the project auxiliary

**Comment by shwang** [comment]
I was a bit sloppy and merged in the codecov ignore changes from master with #298 (forgot that this PR should be merged first). The diff of this PR relative to master now only removes the superfluous ignore.

**Review by AdamGleave — APPROVED** [review]
LGTM

**Comment by AdamGleave** [comment]
> I was a bit sloppy and merged in the codecov ignore changes from master with #298 (forgot that this PR should be merged first).

OK. Might want to consider using something like https://github.com/z0al/dependent-issues in the future to stop accidental merges? (Or just keeping the original branch clean.)

**Pull request merged**